### DISCUSSION TITLE
Piscine CPP, Java ...

### DISCUSSION THREAD

**Discussion by Foohx** [Studio-17/Epitech-Subjects#18] [General]
Bonjour,

Envisagez-vous d'ajouter les exercices des années suivantes ?

Merci d'avance

**Comment by Clement-Fernandes** [comment]
Salut,

Si tu parle de la Tek3 alors bien-sûr ! Ca ne devrait pas trop tarder 😉

**Comment by Foohx** [comment]
Oui je parlais bien de la Tek3 😄
Super ! J'ai hate, et merci beaucoup du partage 👍
